include/freerdp/crypto/crypto.h

```diff
@@ -56,34 +56,41 @@ FREERDP_API CryptoCert crypto_cert_read(BYTE* data, UINT32 length);
 FREERDP_API char* crypto_cert_fingerprint(X509* xcert);
 FREERDP_API char* crypto_cert_subject(X509* xcert);
 FREERDP_API char* crypto_cert_subject_common_name(X509* xcert, int* length);
-FREERDP_API char** crypto_cert_subject_alt_name(X509* xcert, int* count,
```

Comment:
Please do not rename these functions, we try to keep API breakage to a minimum for the remaining changes to 2.0

Reply:
The name was wrong in the first place.

Reply:
@informatimago Well, it might have changed but there are no DNS names in certificates (just a name and alternative names) just like the call to `X509_get_ext_d2i(xcert, NID_subject_alt_name, 0, 0);` suggests.